scripts/convert2PDF.js

```diff
@@ -24,19 +24,31 @@
        return;
      }
    }
+   let file = argv.file;
 
   const readdirPromise = denodeify(fs.readdir);
   let outfile = `Bonita-BPM-documentation`;
 
   const flattenedTaxo = flattenTaxonomy(taxonomy);
   let htmlFile = '', count = 0;  
 
-  rx.Observable.from(flattenedTaxo.filter(fileName => fileName && !fileName.match(/^https?:\/\//)))
-  .select(html => {
-    return 'build/html/' + html + '.html'; 
-  }).bufferWithCount(3).toArray().subscribe(filesArray => {
-    convertLastFilesToPDF(filesArray.reverse());
-  });
+  if (!file) {
+    rx.Observable.from(flattenedTaxo.filter(fileName => fileName && !fileName.match(/^https?:\/\//)))
+    .select(html => {
+      return 'build/html/' + html + '.html'; 
+    }).bufferWithCount(3).toArray().subscribe(filesArray => {
+      convertLastFilesToPDF(filesArray.reverse());
+    });
+  } else {
+    console.log("Generate PDF for file: " + file);
+    rx.Observable.from(flattenedTaxo.filter(fileName => fileName && !fileName.match(/^https?:\/\//) && fileName == file))
+    .select(html => {
+      return 'build/html/' + html + '.html'; 
+    }).bufferWithCount(3).toArray().subscribe(filesArray => {
+      convertLastFilesToPDF(filesArray.reverse());
+    });
```

Comment:
seems to have a lot of duplicated code here.
Would you mind removing some duplication by extracting some method ?
Maybe you can do something like that
```
rx.Observable.from(flattenedTaxo.filter(findAGoodNameForThat)
 .select(html => {
       return 'build/html/' + html + '.html'; 
  }).bufferWithCount(3).toArray().subscribe(filesArray => {
       convertLastFilesToPDF(filesArray.reverse());
  });

function findAGoodNameForThat(file) {
  return function filter(filename){
       // find a better way to do that
      return (file && filename == file) || fileName && !fileName.match(/^https?:\/\//) 
   };
}
```